packages/storybook/.storybook/main.js

```diff
@@ -72,6 +72,17 @@ module.exports = {
       },
     });
 
+    config.plugins.push(
+      new webpack.DefinePlugin({
+        'process.env': {
+          ENABLE_EXPERIMENTAL: JSON.stringify(
+            // Set by Netlify
+            process.env.PULL_REQUEST === 'false' ? 'false' : 'true',
```

Comment:
That means that experimental will only be turned on for Netlify. 
How can we add it locally in a simple way?
How about we use the same approach as the force debug value in useLogger and then we set a global variable?
```
const forceDebug = localStorageAvailable() && window.localStorage.getItem('DEBUG') != null;
```

Reply:
> That means that experimental will only be turned on for Netlify.

@dtassone did you try it locally? In your local env `process.env.PULL_REQUEST` is undefined, so `ENABLE_EXPERIMENTAL` is true.

Reply:
> How about we use the same approach as the force debug value in useLogger and then we set a global variable?

I think that we should go in the other direction, remove the logger for debug and info levels once in production. As far as I understand, there are no use cases for it. However, what we can gain is smaller bunde sizes.